_data/publications/osborne-acat2022-awkward-rdataframe.yml

```diff
@@ -0,0 +1,8 @@
+title: "Awkward to RDataFrame and back"
+link: https://arxiv.org/abs/2302.09860
+date: 2022-10-25
```

Comment:
Hi @ianna - probably best to use the date of the proceedings not the one of the conference here. (eg, Feb 20)

Reply:
however, the proceedings are not published yet...